runtime/src/accounts.rs

```diff
@@ -601,6 +601,7 @@ impl Accounts {
                             feature_set.is_active(&use_default_units_in_fee_calculation::id()),
                             !feature_set.is_active(&remove_deprecated_request_unit_ix::id()),
                             feature_set.is_active(&remove_congestion_multiplier_from_fee_calculation::id()),
+                            feature_set.is_active(&enable_request_heap_frame_ix::id()) || self.accounts_db.expected_cluster_type() != ClusterType::MainnetBeta,
```

Comment:
not super cool to call feature_set and cluster_type everywhere, maybe a followup PR to clean the Load_accounts() param a bit

Reply:
Agreed that we can live with it for this PR - we're already looking up 3 other features immediately above this one. And cluster type just looks to be checking a field.

Reply:
given the number of bank fields passed into `Accounts::load_accounts` (from bank.rs!), it'd make more sense to just pass in the whole bank reference and make `Bank::calulate_fee` a proper method

Reply:
yea, was thinking that direction too. May put in a follow up PR for refactoring.

Reply:
yep, future pr for sure. no need to clutter this set